### PR TITLE
Refinements of load tests for CDS in 1K workloads.

### PR DESCRIPTION
**Pull request by incfly**
- Use new metrics for observing the change.
- More documentation in `README.,md`.
- Do not intend to be fully automated. Manual remove svc-0.

### CODE REVIEW

File: perf/load/pilot/README.md [hunk 1]
```diff
@@ -2,23 +2,49 @@
...
+
+To support large performance testing, many things need to be specially tuned.
+
+- Install Istio version, e.g., 1.1.2.
+- Pilot resource consumption, 6GB memory, 5 replica.
+- Prometheus resource consumption, update to 0.5 cpu, 10GB memory.
+- Prometheus filtering.
+
+  ```yaml
+  source_labels: [ cluster_name ]
```
Comment (mandarjog) [inline perf/load/pilot/README.md]: why is this filtering needed?

Comment (incfly) [reply perf/load/pilot/README.md]: we need metrics `envoy_cluster_upstream_cx_total {cluster_name = 890 || outbound | svc-0.pilot-load.svc.cluster.local`

Comment (incfly) [reply perf/load/pilot/README.md]: however, we can retain everything by just remove this rule, I'll see prometheus having hard time to catch up in that case.

**Comment by incfly** [comment]
Friendly ping, thanks!

**Comment by geeknoid** [comment]
@incfly Do you want to bring this PR over the finish line?

**Comment by incfly** [comment]
@geeknoid sure, I did another pass to resolve the conflicts and `make lint` succeed locally.

**Pull request merged**